src/JiemyuDLL/Dll/Entities/Entity.cs

```diff
@@ -80,13 +80,33 @@ public uint MoveDistance
         }
 
         /// <summary>
-        /// Attacks another entity.
+        /// How far an entity can attack, based on AttackBehavior
         /// </summary>
-        /// <param name="targetEntity"></param>
-        public void Attack(Entity targetEntity)
+        public uint AttackRange
```

Comment:
If there's no special logic on get/set, you can just use "public uint AttackRange {get; set;}"